sdk/attestation/attestation/package.json

```diff
@@ -9,8 +9,9 @@
     "./dist-esm/src/utils/base64.js": "./dist-esm/src/utils/base64.browser.js",
     "./dist-esm/src/utils/blobConvert.js": "./dist-esm/src/utils/blobConvert.browser.js",
     "./dist-esm/src/utils/textEncoding.js": "./dist-esm/src/utils/textEncoding.browser.js",
+    "./dist-esm/src/utils/buffer.js": "./dist-esm/src/utils/buffer.browser.js",
     "./dist-esm/test/utils/base64url.js": "./dist-esm/test/utils/base64url.browser.js",
-    "./dist-esm/test/utils/Buffer.js": "./dist-esm/test/utils/Buffer.browser.js",
+    "./dist-esm/test/utils/buffer.js": "./dist-esm/test/utils/buffer.browser.js",
```

Comment:
do we need two different ones? if they're identical, you can just import the one in src in the one in test

Reply:
THey're completely different. The one in utils handles the Blob->Buffer conversion, the one in test does something I don't understand.

Reply:
Sorry - the test version replaces "Buffer" with "safe-buffer" under node.

Reply:
renaming the test from Buffer to safeBuffer.

Reply:
On second thought, removing the code entirely, it's not actually referenced.

Reply:
Great!